src/randoop/operation/SerializableArrayCreation.java

```diff
@@ -0,0 +1,25 @@
+package randoop.operation;
+
+import java.io.ObjectStreamException;
+import java.io.Serializable;
+
+import randoop.types.TypeNames;
+
+public class SerializableArrayCreation implements Serializable {
```

Comment:
I realize this is a renaming, but can you document it and/or its relationship to ArrayCreation.